content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -28,10 +28,14 @@ Before you begin, you'll need to prepare data files and a Linux instance.
     each webhook to ignore SSL errors within your VCS provider.
   * If you do not have access to a certificate, you can use
     [Let's Encrypt](https://letsencrypt.org/getting-started/) to request one for free.
-    You will provide the path on the host system to the created certificate and key
-    when requested by the setup process.
+    This functionality is available in the settings dashboard, simply click the
+    `Automatically Request Certificate via Let's Encrypt` option.
```

Comment:
noob question - is there a way we can just use Let's Encrypt automatically and not have a user ever think about providing a certificate?

Reply:
We have to ask for an email to use it, but we could default to using it, yes. There are some limits on how many certs can be requested for a domain name in a period of time. I worry if it's on by default, customers accidentally hit the limit.

Reply:
makes sense. I guess it's a question of what is more likely:
- A user hits the Let's Encrypt limit
- A user messes up giving TFE a certificate

The latter has shown to be very likely